app/models/jobseeker.rb

```diff
@@ -1,16 +1,16 @@
 class Jobseeker < ApplicationRecord
   has_encrypted :last_sign_in_ip, :current_sign_in_ip
 
-  devise(*%I[confirmable
-             database_authenticatable
-             lockable
-             omniauthable
-             recoverable
-             registerable
-             timeoutable
-             trackable
-             validatable],
-         omniauth_providers: [:openid_connect])
```

Comment:
This was breaking DFE Sign-in (and hence all those tests errors) as the `omniauthable` module is not compatible with other models using Omniauth :shrug: